packages/dashboard/src/PanelEvent.ts

```diff
@@ -70,6 +70,18 @@ export const PanelEvent = Object.freeze({
 
   // Panel is dropped
   DROPPED: 'PanelEvent.DROPPED',
+
+  // Event to change focus to next stack
+  CYCLE_TO_NEXT_STACK: 'PanelEvent.CYCLE_TO_NEXT_STACK',
+
+  // Event to change focus to previous stack
+  CYCLE_TO_PREVIOUS_STACK: 'PanelEvent.CYCLE_TO_PREVIOUS_STACK',
+
+  // Event to change focus to next tab
+  CYCLE_TO_NEXT_TAB: 'PanelEvent.CYCLE_TO_NEXT_TAB',
+
+  // Event to change focus to previous tab
+  CYCLE_TO_PREVIOUS_TAB: 'PanelEvent.CYCLE_TO_PREVIOUS_TAB',
```

Comment:
I'd prefer you create a new type of event, a `NavigationEvent`. And structure it like `DashboardEvents` instead of how `PanelEvent` is set up, then you can use the methods to wire it up.
`PanelEvent` is for an event occurring on a single panel; `DashboardEvent` is for something related to creating/opening dashboards; `NavigationEvent` seems like it should be separate.